### PR TITLE
IK-372 expression docs added

### PR DESCRIPTION
**Pull request by yashaswi-imagekit**

### CODE REVIEW

File: features/arithmetic-expressions-in-transformations.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+| <p>fs</p> |<ul> <li>fs-bh_mul_0.1</li> <li>fs-bw_div_50</li> </ul> |  |
+
+### Calculation Sequence for multiple operators
+<ol> <li>Multiplication and Division: Multiplication and division are evaluated first from left to right.</li>
+<li>Addition and Subtraction: Addition and subtraction are evaluated next from left to right.</li>
+<li>Modulus: Modulus is evaluated then from left to right.</li>
+<li>Power: Power is evaluated last from left to right.</li></ol>
+
+{% hint style="info" %}
+If you're using expressions inside a layer, then the layer should be a separate chain transformation.
```
Comment (imagekitio) [inline features/arithmetic-expressions-in-transformations.md]: Combine two info box in just one.

File: features/arithmetic-expressions-in-transformations.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+{% endhint %}
+
+### Examples
+To resize the height to half of the original, the width to one-fourth of the original, and then add a yellow color border that's 5% of the current width, you can use the following URL.
+
+URL - [https://ik.imagekit.io/demo/img/plant.jpeg?tr=w-iw_div_4,h-ih_div_2:b-cw_mul_0.05_yellow](https://ik.imagekit.io/demo/img/plant.jpeg?tr=w-iw_div_4,h-ih_div_2:b-cw_mul_0.05_yellow)
+
+![](https://ik.imagekit.io/demo/img/plant.jpeg?tr=w-iw_div_4,h-ih_div_2:b-cw_mul_0.05_yellow)
+
+
```
Comment (imagekitio) [inline features/arithmetic-expressions-in-transformations.md:81]: Why extra new line? It is happening in some other places in this PR as well.

File: features/arithmetic-expressions-in-transformations.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+You can also resize and position the image or video layer relative to the base asset. For example, to first resize the image being overlaid to half of the base image dimensions and then place it one-eight from the top and one-eight from the left, you can use the following URL.
+
+URL - [https://ik.imagekit.io/demo/medium_cafe_B1iTdD0C.jpg?tr=l-image,i-logo-white_SJwqB4Nfe.png,cm-pad_resize,bg-yellow,w-bw_div_2,h-bh_div_2,lx-bw_div_8,ly-bh_div_8,l-end](https://ik.imagekit.io/demo/medium_cafe_B1iTdD0C.jpg?tr=l-image,i-logo-white_SJwqB4Nfe.png,cm-pad_resize,bg-yellow,w-bw_div_2,h-bh_div_2,lx-bw_div_8,ly-bh_div_8,l-end)
```
Comment (imagekitio) [inline features/arithmetic-expressions-in-transformations.md:86]: Need more examples with headings such as text overlay, image overlay, background/border etc.

File: features/arithmetic-expressions-in-transformations.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
+# Arithmetic expressions in transformations
```
Comment (imagekitio) [inline features/arithmetic-expressions-in-transformations.md:1]: Reword
```
ImageKit allows use of arithmetic expressions in certain dimension and position-related parameters, making media transformations more flexible and dynamic. They allow for more flexible and complex transformations without the need to hard code specific values in scenarios where the exact dimensions or positions may not be known beforehand or when they need to be adjusted based on other parameters.

Example use cases

* Dynamic resizing:
  You can use arithmetic expressions to set the dimensions of an image relative to its original dimensions or other parameters. For instance, making the width of an image half of its original size using <ADD EXPRESSION>.

* Positioning overlays:
  Arithmetic expressions can be used to calculate the position of overlays or watermarks based on the dimensions of the original image. For instance, placing a watermark at a position that is 10% from the top and right edges of the image using <ADD EXPRESSION>.

* Adjusting padding in text overlay
  Arithmetic expressions can be used to calculate these values based on other parameters like the image's dimensions or aspect ratio using <ADD EXPRESSION>.
  ```

**Review by imagekitio — CHANGES_REQUESTED** [review]

File: features/arithmetic-expressions-in-transformations.md (hunk 3 above)
Comment (yashaswi-imagekit) [reply features/arithmetic-expressions-in-transformations.md:86]: done

File: features/arithmetic-expressions-in-transformations.md (hunk 2 above)
Comment (yashaswi-imagekit) [reply features/arithmetic-expressions-in-transformations.md:81]: addresed

File: features/arithmetic-expressions-in-transformations.md (hunk 1 above)
Comment (yashaswi-imagekit) [reply features/arithmetic-expressions-in-transformations.md]: done

**Pull request merged**